features/environment.py

```diff
@@ -892,22 +892,28 @@ def clone(self, from_name, cluster_name, to_name):
         }
         self.start(to_name, custom_config=custom_config)
 
+    def backup_restore_config(self, params={}):
+        return {
+            'command': (self.BACKUP_RESTORE_SCRIPT
+                        + ' --sourcedir=' + os.path.join(self.patroni_path, 'data', 'basebackup')).replace('\\', '/'),
+            'test-argument': 'test-value',  # test config mapping approach on custom bootstrap/replica creation
+            **params,
```

Comment:
Lets also make codacy happy:
```suggestion
    def backup_restore_config(self, params=None):
        return {
            'command': (self.BACKUP_RESTORE_SCRIPT
                        + ' --sourcedir=' + os.path.join(self.patroni_path, 'data', 'basebackup')).replace('\\', '/'),
            'test-argument': 'test-value',  # test config mapping approach on custom bootstrap/replica creation
            **(params or {}),
```